libcxx/src/tzdb_list.cpp

```diff
@@ -19,25 +19,25 @@ namespace chrono {
 _LIBCPP_EXPORTED_FROM_ABI tzdb_list::~tzdb_list() { delete __impl_; }
 
 _LIBCPP_NODISCARD_EXT _LIBCPP_EXPORTED_FROM_ABI const tzdb& tzdb_list::front() const noexcept {
```

Comment:
I think it would make sense to also make these public member functions not part of the ABI (by exporting private names instead) in this patch.